### PR TITLE
Add cloud-cli homebrew scripts

### PR DESCRIPTION
**Pull request by zymap**

### CODE REVIEW

File: Formula/cloud-cli.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CloudCli < Formula
+  desc "StreamNative cloud cli tools"
+  homepage "https://streamnative.io/"
+  url "http:/snctl_0.4.1-rc_darwin_amd64.tar.gz"
```
Comment (zymap) [inline Formula/cloud-cli.rb:4]: need to update the url.

File: Formula/cloud-cli.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CloudCli < Formula
```
Comment (EronWright) [inline Formula/cloud-cli.rb:1]: Is the class name significant?

What is the install command that would be used?

Comment (zymap) [reply Formula/cloud-cli.rb:1]: The install command will be `brew install cloud-cli`

Comment (EronWright) [reply Formula/cloud-cli.rb:1]: I would prefer `brew install snctl` since `cloud-cli` is just a repo name.

**Pull request merged**